examples/custom-wgpu-kernel/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+mod backward;
+mod forward;
+
+use burn::tensor::{activation, Tensor};
+
+/// We create a type alias to avoid the ugly disambiguation with associative types.
+pub type FloatTensor<B, const D: usize> = <B as burn::tensor::backend::Backend>::TensorPrimitive<D>;
```

Comment:
This makes me think, the concept of TensorPrimitive is not really explained in the previous sections